srcpkgs/lavalauncher/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'lavalauncher'
+pkgname=lavalauncher
+version=2.0.0
+revision=1
+wrksrc="${pkgname}-v${version}"
+build_style=meson
+hostmakedepends="wayland-devel pkg-config scdoc"
+makedepends="wayland-protocols wayland-devel cairo-devel librsvg-devel"
+short_desc="Simple panel launcher for Wayland"
+maintainer="Jony <maybe-one-day-ubermensch@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://git.sr.ht/~leon_plickat/lavalauncher"
+distfiles="https://git.sr.ht/~leon_plickat/lavalauncher/archive/v${version}.tar.gz"
+checksum=6aa9d6fe189f658dabb709c1902454ebaa7dace8259d39c2256397657a411f9b
+
+case "$XBPS_TARGET_MACHINE" in
```

Comment:
Use `if [ "$XBPS_TARGET_LIBC" = "musl" ]` rather than a case to match `*-musl`.

Reply:
Gotcha!